pandas/compat/pickle_compat.py

```diff
@@ -38,6 +40,11 @@ def load_reduce(self):
                 return
             except TypeError:
                 pass
+        elif args and issubclass(args[0], Tick):
```

Comment:
I assume you added this because one of our pickle tests complained.

Reply:
oh yah.  pickle has been the blocker to `cdef`ing offsets for years

Reply:
kk.

I would add a note to the whatsnew, just in case here. (the issue is that a) its possible that this changes an import of an old pickle to a slower / compat path), and we lose forward compatiblity (which is totally ok), just want to note it.

can put it in api breaking sections. ok with merging this just want to document.

Reply:
will add note in next pass